vignettes/manual/SimpleNBox.Rmd

```diff
@@ -0,0 +1,177 @@
+---
+title: Terrestrial C cycle
+---
+
+Contents:
+
+- [Scientific basis](#simplenbox-science) behind the terrestrial C cycle
+- [Implementation details](#simplenbox-implementation)
+
+# Science {#simplenbox-science}
+
+Hector's land C sink is divided into three pools: vegetation ($C_v$), detritus ($C_d$), and soil ($C_s$).
+These sinks are global by default, but they can be divided into $n$ "biomes" with unique parameters and initial pool sizes.
+
+The total C flux from atmosphere to land at time $t$ ($F_{L}(t)$) is the net of C sequestration by photosynthesis (net primary productivity, $NPP_i(t)$) and C release by heterotrophic respiration ($RH(t)$), summed across each biome $i$:
+
+$$ F_{L}(t) = \sum_{i=1}^{n} \left( NPP_i(t) - RH_i(t) \right) $$
+
+NPP is calculated as the product of a user-specified pre-industrial value ($NPP_0$, default is 50 PgC/year) and a CO~2~ fertilization multiplier ($f(C_{atm}, \beta)$), which in turn is a function of the current atmospheric CO~2~ concentration ($C_{atm}$), the initial ("pre-industrial") CO~2~ concentration ($C_0$), and biome-specific CO~2~ fertilization parameter $\beta_i$ (default = 0.36):
+
+$$ NPP_i(t) = NPP_{0,i} \times f(C_{atm}, \beta_i) $$
+$$ f(C_{atm}, \beta_i) = 1 + \beta_i \left( \log(\frac{C_{atm}}{C_0})\right) $$
+
+Heterotrophic respiration occurs in soil ($RH_s(t)$) and detritus ($RH_d(t))$) as a function of biome-specific atmospheric temperature anomaly ($T_i$, calculated as the global atmospheric temperature anomaly scaled by a biome-specific warming factor $\delta$) and the size of the current soil and detritus C pools ($C_s$ and $C_d$, respectively).
+
+$$ T_i(t) = \delta \times T_G(t) $$
+$$ RH_s = 0.02 C_s Q_{10}^{T_{i}(t) / 10}$$
+$$ RH_d = 0.25 C_d Q_{10}^{T_{i}(t) / 10}$$
+
+Note that the fractions of soil and detritus C available to respiration are hard-coded at 0.02 and 0.25, respectively.
```

Comment:
I think the way to say this is these pools are hard-coded to have base turnover times of 50 and 4 years, respectively.

Reply:
That's not exactly right of course, because of exponential decay. Still, framing it in terms of turnover time or half-life (i.e. 34 and 2 years) is clearer I think.

Reply:
Addressed in b7fd652.